### DISCUSSION TITLE
ObservableProperty编译警告

### DISCUSSION THREAD

**Discussion by boogaloo-jw** [BYJRK/DotNet-Discussions#86] [MVVM]
请问使用了CommunityToolkit  的特性ObservableProperty之后，编译是成功的，程序可以运行！但错误列表里 总提示 “在xxx和xxx之间具有二义性” ，以及代码中的红色波浪线
![image](https://user-images.githubusercontent.com/55581790/233818495-48a85664-6989-4054-b75e-3b7e8354c454.png)

**Comment by BYJRK** [comment]
这确实是一个已有的bug，暂不清楚跟toolkit还是VS有关

**Comment by CrazyZhang666** [comment]
删除obj文件夹，然后重新编译即可
CommunityToolkit 更新到最新版本
